setup.cfg

```diff
@@ -27,6 +27,22 @@ zip_safe = False
 include_package_data = True
 setup_requires = setuptools; setuptools_scm[toml]
 
+
+tests_require =
+    asttokens
+    pytest
+    littleutils
+    rich; python_version >='3.11'
+
+test_suite = tests
```

Comment:
This line isn't needed here.

Reply:
You have marked 4 lines. Which do you mean?
`tests_require` also seems kind of deprecated, but it is probably required for older python versions.

Reply:
I mean `test_suite`. It's also possible that `tests_require` isn't useful here, I don't actually know for sure.

Reply:
ok, I also don't know for sure. lets find out